src/createApiCall.ts

```diff
@@ -72,9 +71,21 @@ export function createApiCall(
     callOptions?: CallOptions,
     callback?: APICallback
   ) => {
-    const thisSettings = settings.merge(callOptions);
-
     let currentApiCaller = apiCaller;
+    const gaxStreamingRetries = (currentApiCaller as StreamingApiCaller)
```

Comment:
nit: if we move this down into the `if (currentApiCaller instanceof StreamingApiCaller)`, then you won't need to typecast.

Reply:
great call